doc/api/url.md

```diff
@@ -1150,3 +1156,5 @@ console.log(myURL.origin);
 [examples of parsed URLs]: https://url.spec.whatwg.org/#example-url-parsing
 [percent-encoded]: #whatwg-percent-encoding
 [stable sorting algorithm]: https://en.wikipedia.org/wiki/Sorting_algorithm#Stability
+[ICU]: intl.html#intl_options_for_building_node_js
```

Comment:
It seems this and the next lines are out of ASCII sorting order used for bottom references.